.github/workflows/backup-cognito-users.yaml

```diff
@@ -0,0 +1,37 @@
+name: Create backup of cognito users
+on:
+  workflow_dispatch:
+env:
+  region: 'eu-west-1'
+  userPool: 'eu-west-1_eYTCV3Nl7'
+jobs:
+  backup-users:
+    name: create backup of cognito users
+    runs-on: ubuntu-20.04
+    steps:
+      - id: setup-node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - id: install
+        name: Install dependencies
+        run: |-
+          npm install -g cognito-backup-restore
+      - id: set-up-creds
+        name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ env.region }}
+          
+      - id: backup-users
+        name: create backup json of cognito users
+        run: |-
+          cbr backup --region ${{env.region}} --key  ${{secrets.AWS_ACCESS_KEY_ID}} --secret ${{ secrets.AWS_SECRET_ACCESS_KEY }} --pool ${{env.userPool}} --directory .
+      - id: save-to-s3
+        name: save backup json to s3
+        run: |-
+          current_date=$(date '+%Y-%m-%d.%H-%M-%S')
+          aws s3api put-object --bucket biomage-backups-production --key ${current_date}/biomage-user-pool-case-insensitive-production/data.json --body ./eu-west-1_eYTCV3Nl7.json
```

Comment:
use environment variable that i set for backup bucket and user pool name here?

Reply:
> 

I added the pool ID as a secret. The region is ok as env variable